Backend/app/dashboard/CommonAPIs/spec/RiderPlatform/Management/API/Customer.yaml

```diff
@@ -0,0 +1,40 @@
+imports:
+  Customer: Dashboard.Common
+  Summary: Dashboard.Common
+
+module: Customer
+
+apis:
+  - GET:
+      endpoint: /list
+      auth: ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS CUSTOMER_LIST
+      query:
+        - limit: Int
+        - offset: Int
+        - enabled: Bool
+        - blocked: Bool
+        - phone: Text
+        - personId: Id Customer
+      response:
+        type: CustomerListRes
+  - DELETE:
+        endpoint: /delete
+        auth: ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS CUSTOMER_DELETE
+        params:
+          customerId: Id Customer
+        response:
+          type: APISuccess
```

Comment:
```
      migrate:
        endpoint: CustomerAPI DeleteCustomerEndpoint
```
where `CustomerAPI DeleteCustomerEndpoint` is old endpoint name (you can check here API.RiderPlatform.Customer)

Reply:
this migration should be applied for all apis except GET